src/ResourceManager/Websites/Commands.Websites.Test/ScenarioTests/WebAppSlotTests.cs

```diff
@@ -122,5 +122,12 @@ public void TestWebAppSwapWithPreviewCompleteSlotSwap()
         {
             WebsitesController.NewInstance.RunPsTest(_logger, "Test-WebAppSwapWithPreviewCompleteSlotSwap");
         }
+
+        [Fact(Skip = "Currently the API fails when adding Azure Storage Accounts for slots. Will enable this test when the API is fixed")]
```

Comment:
We can't merge this PR until the API is fixed

Reply:
@maddieclayton - I will ping you once the API is fixed

Reply:
@vinosoto still waiting for this change